app/locations/none.js

```diff
@@ -0,0 +1,37 @@
+import Ember from 'ember';
+
+const {
+  computed: { reads },
+  inject: { service },
+  get
+} = Ember;
+
+const TEMPORARY_REDIRECT_CODE = 307;
```

Comment:
Making the exact code configurable sounds fine. Didn't want to start an argument about HTTP semantics here - I think that could easily get out of control ;)